test/addon_selection_dialog_test.rb

```diff
@@ -62,6 +70,32 @@
       expect(wrapped_addon.selected?).to eq true
     end
 
+    context "when beta versions are not filtered" do
+      subject(:dialog) { described_class.new(registration) }
+      let(:filter_beta) { false }
+
+      it "sets the filter as 'enabled' in the UI" do
```

Comment:
I know it was my mistake, but we could change it to "sets the filter as not checked in the UI" so it looks more clear. What do you think? Same for the following context.